keystore/filesystem/server_keystore.go

```diff
@@ -766,19 +766,19 @@ func (store *KeyStore) GenerateDataEncryptionKeys(id []byte) error {
 
 // ListKeys enumerates keys present in the keystore.
 func (store *KeyStore) ListKeys() ([]keystore.KeyDescription, error) {
-	privateKeys, err := store.describeDir(store.privateKeyDirectory)
+	keys, err := store.describeDir(store.privateKeyDirectory)
```

Comment:
so we had printed the same entry twice due to omitted check for equal folders for public/private keys?

Reply:
Yes, we listed all keys in the same directory twice